apps/vth-dashboard/src/components/components/navigation.json

```diff
@@ -0,0 +1,14 @@
+{
+  "collectionName": "components_components_navigations",
+  "info": {
+    "displayName": "Navigation"
+  },
+  "options": {},
+  "attributes": {
+    "list": {
+      "type": "component",
+      "repeatable": false,
```

Comment:
This should be repeatable I think? Otherwise you can only have 1 main link.

Reply:
<img width="788" alt="image" src="https://github.com/frameless/strapi/assets/24610692/bcb901fa-4d41-43ad-aa21-c2532857c768">

Reply:
I don't think we need to make the main list repeatable since there will only be one list, right?

![vth-nav](https://github.com/frameless/strapi/assets/53454241/28b1cff1-8700-49b4-9407-6d107acf7d4d)